### DISCUSSION TITLE
Graphing components

### DISCUSSION THREAD

**Discussion by bluisanajmr** [havit/Havit.Blazor#700] [General]
I am using havit Blazer to build a B2B SAS application. The only components that I'm going to have to go to a different vendor for are reporting and graphing.  Has there been any talk of implementing graphing components in havit?  Simply introducing line, pie and bar graphs would be huge.

**Comment by hakenr** [comment]
@bluisanajmr  We use Apache eCharts ourselves. Integration with Blazor is quite simple.
(We also tried JSCharts on some projects, also simple.)

As for your question - No, we are not currently planning a Havit.Blazor library.
If you are interested, we can offer to create a simple wrapper for one of the graph libraries (as a paid consulting service).

**Comment by bluisanajmr** [comment]
Thanks hakenr, 

Loving what you have done with the Havit controls.  I haven't used eCharts before but will check it out.  We are still a few months out from getting into dashboards and reporting but I will let you know if we end up needing help.
